src/app/src/Intro.js

```diff
@@ -1,9 +1,18 @@
 import React, { Component } from 'react';
 
 import { hideIntro } from './app.actions';
+import { showMarkers, updateViewport, setMapStyle } from './map.actions';
+import { mapOverviewViewport, primaryMapStyle } from './constants';
 
 class Intro extends Component {
     render() {
+        const onClick = () => {
+            hideIntro();
+            showMarkers();
+            updateViewport(mapOverviewViewport);
+            setMapStyle(primaryMapStyle);
```

Comment:
Out of curiosity, would it make sense to make one single Redux action out of all of these, then have the Redux listener update the store as necessary all at once rather than sequentially?

Reply:
Yeah, good point. I'll rework this.

Reply:
Actually, going to pass on this for now, due to the fact that some of these are used individually in other parts of the app, and the state of the budget.